src/event_form/steps/event-form-step5.controller.js

```diff
@@ -661,12 +686,33 @@ function EventFormStep5Controller($scope, EventFormData, eventCrud, udbOrganizer
       $scope.bookingInfoCssClass = 'state-complete';
       $scope.savingBookingInfo = false;
       $scope.bookingInfoError = false;
+      removeDuplicateContactBooking();
     }, function() {
       $scope.savingBookingInfo = false;
       $scope.bookingInfoError = true;
     });
   }
 
+  function removeDuplicateContactBooking() {
+    var url = $scope.bookingModel.url;
+    var phone = $scope.bookingModel.phone;
+    var email = $scope.bookingModel.email;
+
+    var foundUrl = $scope.contactInfo.some(function (element) {
```

Comment:
@stijnswaanen Wat is de bedoeling van deze _foundUrl/Phone/Email_ variables? Op het eerste zicht doe je er niets mee.